frontend/src/Dashboard.jsx

```diff
@@ -13,7 +14,7 @@ const Dashboard = () => {
 
     useEffect(() => {
         const callBackendAPI = async () => {
-            const fetchAllURL = "http://localhost:3000/boards";
+            const fetchAllURL = dbApiPath + "/boards";
```

Comment:
i think you should be able to use relative path as well
`const fetchAllURL = "/boards"`